### PR TITLE
Skip vul-severity determination if snyk token is not found

### PR DESCRIPTION
**Pull request by divyansh42**
Signed-off-by: divyansh42 <diagrawa@redhat.com>

### Related Issue(s)
Close : #34 

### Checklist

- [ ] This PR includes a documentation change
- [x] This PR does not need a documentation change
---
- [ ] This PR includes test changes
- [x] This PR's changes are already tested
---
- [ ] This change is a patch change
- [ ] This change is a minor change
- [ ] This change is a major (breaking) change

### CODE REVIEW

File: src/analyse.ts [hunk 1]
```diff
@@ -22,7 +23,7 @@ namespace Analyse {
 
     export async function analyse(
         manifestPath: string, analysisReportName: string,
-    ): Promise<"none" | "warning" | "error"> {
+    ): Promise<"none" | "warning" | "error" | ""> {
```
Comment (tetchel) [inline src/analyse.ts:26]: it's a small thing, but I prefer `undefined` over the empty string here, it's more explicit

File: src/crdaScan.ts [hunk 2]
```diff
@@ -83,12 +83,6 @@ export async function crdaScan(
     ghCore.setOutput(Outputs.REPORT_LINK, reportLink);
 
     if (!crdaData.analysed_dependencies) {
```
Comment (tetchel) [inline src/crdaScan.ts:85]: since we have this same logic in the `analyse` function now, is there some way we could check this only in one place?

Comment (divyansh42) [reply src/crdaScan.ts:85]: I wanted to it like if analyse function returns `""` or `undefined`(now) return from here

Comment (divyansh42) [reply src/crdaScan.ts:85]: I wanted to it like if analyse function returns `""` or `undefined`(now) return from here

Comment (tetchel) [reply src/crdaScan.ts:85]: so the second (outer) `if (!crdaData.analysed_dependencies)` can just be removed i think

Comment (divyansh42) [reply src/crdaScan.ts]: Yes

Comment (divyansh42) [reply src/crdaScan.ts]: Yes

**Comment by divyansh42** [comment]
Close: #34

**Review by tetchel — APPROVED** [review]

**Pull request merged**